projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -49,20 +59,35 @@ export class FacetListComponent {
   @Output() closeList = new EventEmitter();
 
   /** The list of all facet and values related to the products in the list */
-  facetList$: Observable<FacetList> = this.facetService.facetList$;
+  facetList$: Observable<FacetList> = this.facetService.facetList$.pipe(
+    tap((facetList) => {
```

Comment:
I think we should move where we tap this observable instead of where it is instantiated because now it is doing more than just returning its observable. Are we able to tap `this.facetList$` in `ngOnInit`, `ngOnViewInit` or some other place like that?

Reply:
Sure, OnInit will be a better place for that